renv.lock

```diff
@@ -11,45 +11,24 @@
   "Packages": {
     "BH": {
       "Package": "BH",
-      "Version": "1.72.0-3",
+      "Version": "1.75.0-0",
```

Comment:
Packages update should be a individual commit, and it's not the best practice when make a feature commit also update the environment info at the same time.

Reply:
Ah yes sorry about this! Something was happening with the environment and updates and I just went ahead and updated packages without thinking.

Reply:
That's fine! It's not a enterprise level project, so it's OK😉